Controller/Api/CommentController.php

```diff
@@ -84,7 +84,7 @@ public function getCommentAction($id)
      *
      * @param int $id A comment identifier
      *
-     * @return View
+     * @return \FOS\RestBundle\View\View
```

Comment:
I would add a use for this other View with an alias at line 98 it could be useful. WDYT?